modules/archiving-a-model-version.adoc

```diff
@@ -0,0 +1,37 @@
+:_module-type: PROCEDURE
+
+[id="archiving-a-model-version_{context}"]
+= Archiving a model version
+
+[role='_abstract']
+You can archive a model version. The model version will be archived and unavailable for use unless it is restored.
```

Comment:
[Would be helpful to add] _You can archive a model version that you no longer require_, or similar.

Reply:
Updated.